docs/tutorials/py-config-interpreter.md

```diff
@@ -47,8 +47,8 @@ To set the interpreter, you can use the `interpreter` configuration in the `py-c
   <body>
     <py-config>
       [[interpreters]]
-        src = "https://cdn.jsdelivr.net/pyodide/v0.22.0a3/full/pyodide.js"
-        name = "pyodide-0.22.0a3"
+        src = "https://cdn.jsdelivr.net/pyodide/v0.23.0/full/pyodide.js"
+        name = "pyodide-0.23.0"
```

Comment:
shouldn't this be 0.23.2 too?

Reply:
My thinking is that, since the purpose of this tutorial is setting a different version of the interpreter and being able to verify that it has happened, that this should actually not be the default version. I'll add a sentence in that section clarifying this.